tests/test_config_contacts.cpp

```diff
@@ -443,10 +451,428 @@ TEST_CASE("huge contacts compression", "[config][compression][contacts]") {
 
     auto [seqno, to_push, obs] = contacts.push();
     CHECK(seqno == 1);
-    CHECK(to_push.size() == 46'080 + 181);  // 181 == protobuf overhead
+    CHECK(to_push.size() == 1);
+    CHECK(to_push[0].size() == 56'320 + 181);  // 181 == protobuf overhead
     auto dump = contacts.dump();
-    // With tons of duplicate info the push should have been nicely compressible:
-    CHECK(dump.size() > 1'320'000);
+    // With tons of duplicate info the push should have been nicely compressible, but our dump
+    // (which currently isn't compressed) is much larger:
+    CHECK(dump.size() == EXPECT_BIG_DUMP_SIZE);
+
+#if 0
```

Comment:
What are the `#if 0` for? (just debugging the tests?)

Reply:
Yeah, this is debugging that can go.

Reply:
Also `#if 0` is a proprocessor directive so that unlike `if (0) { ... }` the `...` doesn't have to compile.  (And it wouldn't here, because I deleted the iostream header for std::cerr).

Reply:
Fixed.